Source/WebCore/editing/FrameSelection.h

```diff
@@ -144,10 +144,10 @@ class FrameSelection final : private CaretBase, public CaretAnimationClient {
     WEBCORE_EXPORT void moveTo(const VisiblePosition&, const VisiblePosition&, UserTriggered = UserTriggered::No);
     void moveTo(const Position&, Affinity, UserTriggered = UserTriggered::No);
     void moveTo(const Position&, const Position&, Affinity, UserTriggered = UserTriggered::No);
-    void moveWithoutValidationTo(const Position&, const Position&, bool selectionHasDirection, bool shouldSetFocus, SelectionRevealMode, const AXTextStateChangeIntent& = AXTextStateChangeIntent());
+    void moveWithoutValidationTo(const Position&, const Position&, bool selectionHasDirection, bool shouldSetFocus, SelectionRevealMode, const AXTextStateChangeIntent& = AXTextStateChangeIntent(), bool isForBindings = false);
```

Comment:
Other than adding a new boolean, we should probably add a new value of `SetSelectionOption` instead.
e.g. `SetSelectionOption::IsScriptTriggered` or `SetSelectionOption::ForBindings`